services/gundeck/src/Gundeck/Push.hs

```diff
@@ -276,13 +276,18 @@ pushViaRabbitMq p = do
   for_ routingKeys $ \routingKey ->
     mpaPublishToRabbitMq userNotificationExchangeName routingKey qMsg
 
-pushAllToCells :: (MonadPushAll m) => [Push] -> m ()
+pushAllToCells :: (MonadPushAll m, Log.MonadLogger m) => [Push] -> m ()
 pushAllToCells ps = do
   mQueue <- mpaCellsEventQueue
-  for_ mQueue $ \queue -> for_ ps (pushToCells queue)
+
+  let cellPushes = filter (._pushIsCellsEvent) ps
+  unless (null cellPushes) $ case mQueue of
+    Nothing -> do
+      Log.err $ msg ("Cells events have been generated, but no Cells queue is configured in gundeck" :: ByteString)
+    Just queue -> traverse_ (pushToCells queue) cellPushes
 
 pushToCells :: (MonadPushAll m) => Text -> Push -> m ()
-pushToCells queue p = when p._pushIsCellsEvent $ do
+pushToCells queue p = do
```

Comment:
what's behind this change?

Reply:
I added a log message that should help diagnose configuration mistakes (e.g. forgetting to set the cells queue name). With this, we generate a single log message rather than one per push.